src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1040,19 +1040,16 @@ uct_dc_mlx5_ep_fc_pure_grant_send_completion(uct_rc_iface_send_op_t *send_op,
     uct_dc_fc_request_t *fc_req = (uct_dc_fc_request_t*)send_op->buffer;
     uct_dc_mlx5_ep_t *fc_ep     = ucs_derived_of(fc_req->super.ep,
                                                  uct_dc_mlx5_ep_t);
-    char gid_str[32];
+    char buf[128];
 
     if (ucs_likely(!(send_op->flags & UCT_RC_IFACE_SEND_OP_STATUS) ||
                    (send_op->status != UCS_ERR_CANCELED))) {
         /* Pure grant sent - release it */
         ucs_mpool_put(fc_req);
     } else {
-        ucs_trace("fc_ep %p: re-sending FC_PURE_GRANT (seq:%" PRIu64 ")"
-                  " to dct_num:0x%x, lid:%d, gid:%s",
-                  fc_ep,  fc_req->sender.payload.seq, fc_req->dct_num,
-                  fc_req->lid,
-                  uct_ib_gid_str(ucs_unaligned_ptr(&fc_req->sender.payload.gid),
-                                 gid_str, sizeof(gid_str)));
+        ucs_trace("fc_ep %p: re-sending %s", fc_ep,
+                  uct_dc_mlx5_iface_fc_pure_grant_str(
+                          fc_req, buf, ucs_static_array_size(buf)));
```

Comment:
sizeof(buf) for consistency with other code

Reply:
done